src/serac/numerics/tests/odes.cpp

```diff
@@ -234,7 +235,8 @@ int which_kind_of_ode(serac::TimestepMethod m)
   if (m == serac::TimestepMethod::CentralDifference) return 2;
   if (m == serac::TimestepMethod::FoxGoodwin) return 2;
 
-  return -1;
+  SLIC_ERROR(axom::fmt::format("Unsupported serac::TimestepMethod {0}", static_cast<int>(m)));
+  return 0;
```

Comment:
Clang-tidy complained this was returning -1, because it figured out the return value was being used as an index. I can see we'd still want to fail in this case, so I decided to explicitly print an error instead.